test_collections/matter/sdk_tests/support/python_testing/models/test_case.py

```diff
@@ -254,10 +250,7 @@ async def execute(self) -> None:
         try:
             logger.info("Running Python Test: " + self.python_test.name)
 
-            BaseManager.register("TestRunnerHooks", SDKPythonTestRunnerHooks)
-            manager = BaseManager(address=("0.0.0.0", 50000), authkey=b"abc")
-            manager.start()
-            test_runner_hooks = manager.TestRunnerHooks()  # type: ignore
+            test_runner_hooks = self.sdk_container.manager.TestRunnerHooks()  # type: ignore
```

Comment:
```suggestion
            test_runner_hooks = self.sdk_container.manager.TestRunnerHooks()  # type: ignore # flake8: noqa E501
```
Could try this in order to ignore `Line too long (92 > 88 characters) (E501)`